docs/1.introduction.md

```diff
@@ -0,0 +1,12 @@
+## Introduction
```

Comment:
Hi Louise - yes I think that would make a lot of sense. Right now the introduction is quite short anyway and it would be good to provide some additional context on the landing page.